lib/client/api_login_test.go

```diff
@@ -138,13 +140,15 @@ func TestTeleportClient_Login_local(t *testing.T) {
 
 	ctx := context.Background()
 	tests := []struct {
-		name             string
-		secondFactor     constants.SecondFactorType
-		inputReader      *prompt.FakeReader
-		solveWebauthn    func(ctx context.Context, origin string, assertion *wanlib.CredentialAssertion, prompt wancli.LoginPrompt) (*proto.MFAAuthenticateResponse, error)
-		authConnector    string
-		allowStdinHijack bool
-		preferOTP        bool
+		name                 string
+		secondFactor         constants.SecondFactorType
+		inputReader          *prompt.FakeReader
+		solveWebauthn        func(ctx context.Context, origin string, assertion *wanlib.CredentialAssertion, prompt wancli.LoginPrompt) (*proto.MFAAuthenticateResponse, error)
+		authConnector        string
+		allowStdinHijack     bool
+		preferOTP            bool
+		isPswdlessRegistered bool
+		attachment           wancli.AuthenticatorAttachment
```

Comment:
```suggestion
		authenticatorAttachment           wancli.AuthenticatorAttachment
```

so it's a bit clearer?